src/git-duet/git-duet-rebase/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("rebase"))
```

Comment:
I'm missing some context of how git-duet works under the covers - I asked some specific questions on #4.

If you could answer those, I'll be able to give a more informed answer here :)